src/theme.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { DEFAULT_THEME, PALETTE } from '@zendeskgarden/react-theming';
+import { rgba } from 'polished';
+
+const BASE = 4;
+
+export const theme = {
+  screens: {
+    sm: DEFAULT_THEME.breakpoints.sm,
+    md: DEFAULT_THEME.breakpoints.md,
+    lg: DEFAULT_THEME.breakpoints.lg,
+    xl: DEFAULT_THEME.breakpoints.xl
+  },
+  colors: {
+    transparent: 'transparent',
+    ...PALETTE
+  },
+  spacing: {
+    px: '1px',
+    0: '0',
+    1: `${BASE}px`,
+    2: `${BASE * 2}px`,
+    3: `${BASE * 3}px`,
+    4: `${BASE * 4}px`,
+    5: `${BASE * 5}px`,
+    6: `${BASE * 6}px`,
+    7: `${BASE * 7}px`,
+    8: `${BASE * 8}px`,
+    9: `${BASE * 9}px`,
+    10: `${BASE * 10}px`,
+    12: `${BASE * 12}px`,
+    16: `${BASE * 16}px`,
+    20: `${BASE * 20}px`,
+    24: `${BASE * 24}px`,
+    32: `${BASE * 32}px`,
+    40: `${BASE * 40}px`,
+    48: `${BASE * 48}px`,
+    56: `${BASE * 56}px`,
+    64: `${BASE * 64}px`
+  },
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
+  borderColor: (localTheme: any): any => ({
+    ...localTheme('colors'),
+    default: localTheme('colors.grey.300', 'currentColor')
+  }),
+  borderRadius: {
+    none: '0',
+    sm: DEFAULT_THEME.borderRadii.sm,
+    default: DEFAULT_THEME.borderRadii.md,
+    full: '9999px'
+  },
+  borderWidth: {
+    default: DEFAULT_THEME.borderWidths.sm,
+    0: '0',
+    2: '2px',
+    3: '3px',
```

Comment:
Should be `DEFAULT_THEME.borderWidths.md`. Why do we want the other values (2, 4, 8) in here? There's nothing in the system that corresponds.

Reply:
Within tailwind the [border width utilities are linear](https://tailwindcss.com/docs/border-width/#app), not t-shirt sized. Since these are pixel-based `default-1px`, `2->2px`, `4->4px` I've brought over the default Tailwind values to align with their documentation.

Reply:
Sticking to the default Tailwind format has been possible in the other theme areas. To help with documentation it would be helpful if we could align here as well.

Reply:
Ok. I was thinking you could use `3: DEFAULT_THEME.borderWidths.md`, but I guess that has the chance of leading to future unpredictability.